rosdoc2/verbs/build/builders/sphinx_builder.py

```diff
@@ -15,21 +15,41 @@
 import json
 import logging
 import os
+import setuptools
 import shutil
 import subprocess
+import sys
 
 from ..builder import Builder
 from ..collect_inventory_files import collect_inventory_files
 from ..create_format_map_from_package import create_format_map_from_package
 
 logger = logging.getLogger('rosdoc2')
 
+def generate_package_toc_entry(*, build_context) -> str:
+    build_type = build_context.build_type
+    always_run_doxygen = build_context.always_run_doxygen
+    always_run_sphinx_apidoc = build_context.always_run_sphinx_apidoc
+    toc_entry_py = f'\n   {build_context.package.name} Python API <modules>'
+    toc_entry_cpp = \
+        '\n   api/library_root\n   Full C/C++ API <api/unabridged_api> \
+        \n   File structure <api/unabridged_orphan>'
```

Comment:
I'd recommend using a multi-line string for this, it's a bit easier to read.

```suggestion
    toc_entry_cpp = """
   api/library_root
   Full C/C++ API <api/unabridged_api>
   File structure <api/unabridged_orphan>"""
```

Also did you mean for it to be indented by three spaces inside the string? Maybe that's right for the rst content, but I just wanted to check, since that's kind of odd.